app/views/follow_requests/_follow_unfollow.html.erb

```diff
@@ -1,17 +1,18 @@
-<% unless sender == recipient %>
-  <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
+    <% unless sender == recipient %>
+    <% follow_request = current_user.sent_follow_requests.find_by(recipient: recipient) %>
 
-  <% if follow_request %>
-    <% if follow_request.pending? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
-        Un-request
-      <% end %>
-    <% elsif follow_request.accepted? %>
-      <%= link_to follow_request,  data: { turbo_method: :delete }, class: "btn btn-outline-secondary" do %>
-        Un-follow
+<% if follow_request %>
+      <% if follow_request.pending? %>
+        <%= link_to follow_request,  method: :delete, remote: true, class: "#{dom_id(follow_request)} btn btn-outline-secondary" do %>
```

Comment:
I like that you created a class for both "un-follow" and "un-request" to create functionality for these separate link to's!